components/brave_wallet/browser/password_encryptor_unittest.cc

```diff
@@ -16,7 +17,7 @@ namespace brave_wallet {
 
 namespace {
 base::span<const uint8_t> ToSpan(std::string_view sv) {
-  return base::as_bytes(base::make_span(sv));
+  return base::as_byte_span(sv);
 }
 std::string ToString(const std::vector<uint8_t>& v) {
   return std::string(v.begin(), v.end());
```

Comment:
If possible, consider deleting this `ToString` function and just use `base::as_string_view(base::as_chars(v))`

Reply:
https://github.com/brave/brave-core/pull/26431/commits/6809115eb5276f7e5f97305fc0e0189ade29d515